src/generic/srchctlg.cpp

```diff
@@ -460,12 +466,11 @@ wxSize wxSearchCtrl::DoGetBestClientSize() const
         cancelMargin = FromDIP(MARGIN);
     }
 
-    int horizontalBorder = FromDIP(1) + ( sizeText.y - sizeText.y * 14 / 21 ) / 2;
+    const int horizontalBorder = (sizeText.y - sizeText.y * 14 / 20) / 2;
```

Comment:
Do you have any explanation for the changes in this line?

Reply:
This is the most important part: in `void wxSearchCtrl::LayoutControls()` calculations:
**horizontalBorder**: here I added const because it holds the border sizes. as for `(sizeText.y - sizeText.y * 14 / 20) / 2;`: I don't know why adding FromDIP(1) to the border calculation. And the number 21 does not appear in any line of the entire code. only (20).